README.md

```diff
@@ -128,6 +128,18 @@ NAME:
      --help, -h  show help
 ```
 
+### Setting an environment file
+
+Testground automatically loads an `.env.toml` file at root of your source
```

Comment:
Don't forget to add the .env.toml to `.gitignore` or many keys will be leaked by mistake :)

Reply:
It was already there since the beginning 😛

Reply:
Ah! I missed it (was reviewing only the commit). Thanks ! :)